temba/ussd/models.py

```diff
@@ -37,7 +37,8 @@ def start_session_async(self, flow, urn, content, date, message_id):
         message = Msg.objects.create(
             channel=self.channel, contact=self.contact, contact_urn=self.contact_urn,
             sent_on=date, session=self, msg_type=USSD, external_id=message_id,
-            created_on=timezone.now(), modified_on=timezone.now(), org=self.channel.org)
+            created_on=timezone.now(), modified_on=timezone.now(), org=self.channel.org,
+            direction=self.USSD_PULL)
```

Comment:
I'm fine with `Msg.INCOMING`, just defaulted to `USSD_PULL` because it's the attribute that's set and used in the [`USSDSession` model](https://github.com/praekeltfoundation/rapidpro/blob/3e7abec4386518a28d683307f1f1c739e08e7616/temba/ussd/models.py#L27)